public/data/cchow33.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Carmen Chow",
+  "type": "personal",
+  "bio": "Aspiring FE Dev!",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/cchow33",
+      "icon": "github"
+    },
+  ],
+  "milestones": [
+    {
+      "title": "Decided to join a bootcamp",
+      "date": "October 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Looking forward to accelerating my learning!",
```

Comment:
Oops. There seems to be a comma at the end of line 18 that is not required. Feel free to remove it :smiley: 
```suggestion
      "description": "Looking forward to accelerating my learning!"
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix